tests/_data/fixtures/Di/services.php

```diff
@@ -1,15 +1,19 @@
 <?php
 
+use Phalcon\Config;
+use Phalcon\Test\Fixtures\Di\SomeComponent;
+use Phalcon\Test\Module\UnitTest;
+
 return [
     'unit-test' => [
-        'className' => \Phalcon\Test\Module\UnitTest::class,
+        'className' => UnitTest::class,
     ],
     'config' => [
-        'className' => \Phalcon\Config::class,
-        'shared' => true,
+        'className' => Config::class,
+        'shared'    => true,
     ],
     'component' => [
-        'className' => \SomeComponent::class,
+        'className' => SomeComponent::class,
```

Comment:
We needed that to be a single class that has been `required` hence no namespace for this.